README.md

```diff
@@ -120,21 +162,24 @@ repositories {
 }
 
 dependencies {
-    implementation ("com.gravatar:gravatar:<version>")
+    implementation("com.gravatar:gravatar:<version>")
     // OR
-    implementation ("com.gravatar:gravatar-ui:<version>")
+    implementation("com.gravatar:gravatar-ui:<version>")
 }
 ```
 
 ## Generating the API documentation
 
-We're using [kdoc](https://kotlinlang.org/docs/kotlin-doc.html) to document the library's code. [Dokka](https://kotlinlang.org/docs/dokka-introduction.html) has been setup to generate html documentation from kdoc. To generate the HTML docs in `docs/dokka/`, run the following command:
+We're using [kdoc](https://kotlinlang.org/docs/kotlin-doc.html) to document the library's
+code. [Dokka](https://kotlinlang.org/docs/dokka-introduction.html) has been setup to generate html documentation from kdoc. To generate the
+HTML docs in `docs/dokka/`, run the following command:
 
 ```sh
 ./gradlew dokkaHtmlMultiModule
 ```
 
-Here is the [Latest Gravatar-SDK-Android API documentation generated by Dokka](https://automattic.github.io/Gravatar-SDK-Android/trunk/index.html)
+Here is
+the [Latest Gravatar-SDK-Android API documentation generated by Dokka](https://automattic.github.io/Gravatar-SDK-Android/trunk/index.html)
```

Comment:
Super nitpicking, but if we go that route and limit line width on markdown docs, we should maybe switch to [reference-style links][1]

[1]: https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
I didn't want to do that 🤦🏼‍♂️. I use the IDE formatting before commiting... my bad. 

Updated.